idd/Energy+.idd.in

```diff
@@ -52999,9 +53001,13 @@ AirLoopHVAC:UnitarySystem,
        \type choice
        \key Load
        \key SetPoint
+       \key ASHRAE90VariableFan
```

Comment:
I won't hold anything up if this key has already been approved during calls, but I will say that, to me, `ASHRAE90VariableFan` doesn't have a good feel to it.  Maybe `SingleZoneVAV`?  `SingleZoneVariable`?